website/docs/table-group/data-table/data-table-a11y.md

```diff
@@ -3,20 +3,55 @@ title: DataTable
+| --------------------------------- | --------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `DataTable`                       | `grid`                                  | Identifies the element that serves as the grid widget container. The `grid` element isn't focusable because it implements the practice described in the section on [Managing Focus Within Components Using a Roving tabindex](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#kbd_roving_tabindex). |
+|                                   | `aria-rowcount`                         | Defines the total number of rows in the `grid`.                                                                                                                                                                                                                                                                   |
+|                                   | `aria-colcount`                         | Defines the total number of columns in the `grid` when not all columns are present in the DOM.                                                                                                                                                                                                                    |
+| `DataTable.Head`, `DataTable.Row` | `row`                                   | Identifies a row of cells within a tabular structure. A `row` contains one or more grid cells or column headers, and possibly a row header, within a `grid`.                                                                                                                                                      |
+|                                   | `aria-rowindex="INDEX_VALUE"`           | Defines an element's position with respect to the total number of rows within the `grid`.                                                                                                                                                                                                                         |
+| `DataTable.Head`                  | `aria-sort="ascending/descending/none"` | This attribute is set to a specific value when a column is sorted in ascending or descending order, or when it’s sortable but not yet sorted.                                                                                                                                                                     |
```

Comment:
```suggestion
| `DataTable.Head`                  | `aria-sort="ascending/descending"` | This attribute is set to a specific value when a column is sorted in ascending or descending order.                                                                                                                                                                     |
```
It seems that aria-sort=none isn't supported by anything as an indicator that the column is **_sortable_**. I'll double check tomorrow, but I guess we should remove the "sortable" mention.